gsplat/experimental/cuda/_warpper.py

```diff
@@ -0,0 +1,588 @@
+from typing import Callable, Tuple
```

Comment:
nit: rename file to `gsplat/experimental/cuda/_wrapper.py`?

Reply:
oh great catch!